### ISSUE TITLE
[REFACTOR] MOLECULES adjust Storefront UI design system

### ISSUE THREAD

**Issue by pspaczek** [DivanteLtd/storefront-ui#805]
rewrite molecules CSS for a new design system http://bit.ly/storefront-ui-design-system

Workflow:
1. create a branch from `refactor/design-system`
2. create PR with your changes

Notes: 
1. **don't** touch components props API
2. note on PR all changes in component CSS modifiers API
3. in components styles you should use **always** css variables, don't use hardcode values for spacers, colour's, font's etc.
4. spacers should be align to 8px grid like on Google Matrial
5. refactor workflow
- move css to outide file
- set style for base component
a. set font
b. set spacers
c. set color's
d. set others property
- create modifiers
- test it 🙏
- push it
6. we shouldn't use different element font size in component, it should be the same
``` scss
// *don't
.sf-button{
font-size: --body-font-size-regular;
  @include for-desktop {
    font-size: --body-font-size-extra-big;
  }
}
```
7.  scss filer structure
``` scss
@import "../../helpers";
.sf-button{
  width: 
  height:
  padding:
  @include for-desktop {
  --button-padding:
  }
  &--modifier {
    --button-padding:
   }
}
```
Scope of work:
*please assigne you to component 
- [ ] Add to cart @justyna-13
- [x] Alert @aniamusial 
- [x] Banner @AdamPawlinski 
- [ ] Bar @justyna-13 
- [x] Call to action @AdamPawlinski 
- [ ] Characteristic @justyna-13 
- [x] Filter @AdamPawlinski 
- [ ] Gallery @pspaczek 
- [x] Menu Item @aniamusial 
- [x] Modal @aniamusial 
- [x] Notification @martaradziszewska 
- [x] Pagination @martaradziszewska 
~~[ ] Product option @aniamusial~~ TO BE REMOVED
- [x] Radio @martaradziszewska 
- [x] Review @martaradziszewska 
- [x] Scrollable @aniamusial 
- [x] Search bar @aniamusial 
- [x] Section @martaradziszewska 
- [ ] Select @martaradziszewska 
- [ ] Sliding Section @martaradziszewska 
- [ ] Steps @justyna-13
- [ ] Sticky

in refactor/design-system:
- [ ] Add to cart @justyna-13
- [x] Alert @aniamusial 
- [ ] Banner @AdamPawlinski 
- [x] Bar @justyna-13 
- [ ] Call to action @AdamPawlinski 
- [ ] Characteristic @justyna-13 
- [ ] Filter @AdamPawlinski 
- [ ] Gallery @pspaczek 
- [x] Menu Item @aniamusial 
- [ ] Modal @aniamusial 
- [ ] Notification @martaradziszewska 
- [ ] Pagination @martaradziszewska 
~~[ ] Product option @aniamusial~~ TO BE REMOVED
- [ ] Radio @martaradziszewska 
- [ ] Review @martaradziszewska 
- [ ] Scrollable @aniamusial 
- [ ] Search bar @aniamusial 
- [ ] Section @martaradziszewska 
- [ ] Select @martaradziszewska 
- [ ] Sliding Section @martaradziszewska 
- [ ] Steps @justyna-13
- [ ] Sticky

**Comment by justyna-13** [comment]
Wezmę Bar

**Comment by justyna-13** [comment]
Wezmę Characteristic

**Comment by AdamPawlinski** [comment]
Wezmę Banner i Call to Action

**Comment by pspaczek** [comment]
merged all atoms to #871
